### PR TITLE
Increase padding bottom figure and paragraph

### LINKED ISSUE #379: Images and Text should have 24px padding above & below
**Is your feature request related to a problem? Please describe.**
Issue raised by Natalie (UX). Our Text and Image blocks should have 24px of padding above & below. Not clear if that means 12px in reality (12px padding-bottom from first block, plus 12px padding-top from second block).

"Needs an extra 8px".

**Describe the solution you'd like**
A PR to increase the padding as per designs. 

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Snapshots need updating. Work with UX to review the designs in storybook once updated.

**Additional context**
N/A

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #379

**Overall change:** _Increases the padding-bottom of psammead-figure and psammead-paragraph from 16px to 24px_

**Code changes:**

- _Changes psammead-figure's and  psammead-paragraph's padding-bottom to use GEL_SPACING_TRIPLE_
- _Updates the snapshots_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by pjlee11** [comment]
## Whole page screenshot 

padding-bottom-24px

---

## Image screenshot with padding-bottom

image

---

## Text screenshot with padding-bottom

image

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 | Version | Description |
 |---------|-------------|
+| 1.1.1   | [PR#379](https://github.com/BBC/psammead/pull/379) Updates for psammead-figure and psammead-paragraph |
```
Comment (ChrisBAshton) [inline CHANGELOG.md]: This PR doesn't change Psammead itself. We should avoid bumping the version of the root.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (ChrisBAshton) [inline package.json]: 👎

**Review by ChrisBAshton — COMMENTED** [review]
Code looks good apart from I disagree we should be bumping the root level version number.

Also, wasn't our solution going to be more generic (a global padding increase rather than on specific components...?) If that's not possible, fine 👍 thanks!

File: package.json (hunk 2 above)
Comment (pjlee11) [reply package.json]: Happy to leave as is

**Comment by pjlee11** [comment]
I'm going to hold off on this as I think https://github.com/bbc/psammead/issues/394 should be a precursor

File: CHANGELOG.md (hunk 1 above)
Comment (pjlee11) [inline CHANGELOG.md]:
```suggestion

```

**Comment by pjlee11** [comment]
@ChrisBAshton having spoke with UX we will continue to define the bottom spacing on a component. For some components we will defined spacing above and below, when doing this we will need to explore the edge cases and get those signed off by UX. 

Off the back of this PR I explored collapsible margins however they do not work with grid 😞

**Review by dr3 — DISMISSED** [review]

**Review by ChrisBAshton — DISMISSED** [review]
Code looks good - let's have a UX review!

**Review by dr3 — APPROVED** [review]

**Review by sareh — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**